lesson02/assignment/css/_settings.variables.scss

```diff
@@ -9,4 +9,6 @@ $orange: #ff6600;
 $yellow: #ffb617;
 $robins: #01c1d6;
 $purple: #6e27c5;
-$blue: #150f96;
\ No newline at end of file
+$blue: #150f96;
+$bkgrdclr: #f5f5f5;
+$brdrclr: #f6f5f6; // doesn't work but $green does ??
```

Comment:
This should work. You might have a compilation issue in your gulp process. Make sure your CSS file is being regenerated each time you change the code. We can review together to ensure there's not errors preventing this from working.